### PR TITLE
Bias correction for exponential moving average

### PR DESCRIPTION
**Pull request by kuke**
When applying EMAs to parameters, they should be divided by a factor for zero debias.

### CODE REVIEW

**Comment by kuke** [comment]
image

File: python/paddle/fluid/optimizer.py [hunk 1]
```diff
@@ -2175,19 +2176,41 @@ class ExponentialMovingAverage(object):
 
     ..  math::
 
-	\\text{EMA}_t = \\text{decay} * \\text{EMA}_{t-1} + (1 - \\text{decay}) * \\theta_t
+        \\text{EMA}_0 & = 0
 
- 
-    The average results will be saved in temporary variables which can be
```
Comment (xsrobin) [inline python/paddle/fluid/optimizer.py:2181]: LGTM

**Review by xsrobin — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]
LGTM.

**Review by luotao1 — APPROVED** [review]
LGTM for doc

**Review by shanyi15 — APPROVED** [review]

**Pull request merged**